manifest: Clear srp implementation (#14304)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

    In |NCS| Matter samples, the default reaction to migration failure is a factory reset of the device.
    To change the default reaction, set the :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_FACTORY_RESET_ON_KEY_MIGRATION_FAILURE` Kconfig option to ``n``.

>   * Factory reset sequence now can remove all srp client services, of the device, established with the OTBR.

Reviewer: ```suggestion
  * Factory reset sequence can now remove all SRP Client services of the device that have been established with the OTBR.
```

>   * Factory reset sequence now can remove all srp client services, of the device, established with the OTBR.

Reviewer: This one's good too.

>     This ensures that there is no "old" device's DNS entries that can be mistakenly discovered.

Reviewer: ```suggestion
    This ensures that there are no DNS entries for old devices that could be mistakenly discovered.
```

Reviewer: Oh, I like that. I don't think it needs the words "present", but otherwise much clearer on what is meant.
